notebooks/MEMA_episode_visualisation.ipynb

```diff
@@ -10,14 +10,18 @@
   },
```

Comment:
<p>Shouldn't we suppress this notebook ? It does not have any purpose nowadays and is no longer functionnal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
done